netctl/netctl/main.go

```diff
@@ -2,16 +2,25 @@ package main
 
 import (
 	"os"
+	"sort"
 
 	"github.com/codegangsta/cli"
 	"github.com/contiv/netplugin/netctl"
 	"github.com/contiv/netplugin/version"
 )
 
+type byName []cli.Command
+
+func (a byName) Len() int           { return len(a) }
+func (a byName) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a byName) Less(i, j int) bool { return a[i].Name < a[j].Name }
+
 func main() {
 	app := cli.NewApp()
 	app.Flags = netctl.NetmasterFlags
 	app.Version = "\n" + version.String()
+	// use sort.Slice() in go1.8
```

Comment:
@rchirakk Yeah, that makes sense.  I might even make it a `TODO` so it's easier to find.

Based on the [bugs being fixed for 1.8.2](https://github.com/golang/go/milestone/54), it seems like 1.8.1 was an okay release